app/handlers/dbindexes.py

```diff
@@ -261,7 +262,7 @@ def _ensure_stats_indexes(database):
         [(models.CREATED_KEY, pymongo.DESCENDING)], background=True)
 
 
-def _ensure_regressions_indexes(database):
+def _ensure_boot_regressions_indexes(database):
     """Ensure indexes exist on the regression collection.
```

Comment:
The docstring should be updated with "on the boot regressions collection" to match.